crossing-training.lic

```diff
@@ -382,15 +382,36 @@ class CrossingTraining
   def calculate_mana(min, more, discern_data, cyclic)
     total = min + more
     total = (total * @settings.prep_scaling_factor).floor
-    discern_data['mana'] = [(total / 4.0).ceil, min].max
+    discern_data['mana'] = [(total / 5.0).ceil, min].max
     remaining = total - discern_data['mana']
-    if remaining > @settings.cambrinth_cap
+    total_cambrinth_mana = 0
+    @settings.cambrinth_items.each do |item|
+      total_cambrinth_mana += item[:cap]
+    end
+    if remaining > total_cambrinth_mana && !!@settings.cambrinth_items
```

Comment:
I guess it doesn't.  For some reason I thought if I was checking for true/false, it needed to explicitly be a boolean, which is what I was doing with `!!`.  Not needed, though -- updated to remove.